site/content/dev-guide/how-tos/implement-an-adapter.md

```diff
@@ -0,0 +1,112 @@
++++
+title="Implement an Adapter"
+weight=56
++++
+
+NoSqlBench is composed of a sound architectural underpinning that makes the development and addition of new adapters as seamless and quick a process as possible. nb provides a hierarchy of classes which can be extended and interfaces to be implemented, in order to avoid having to re-implement the application infrastructure and in so doing reduces both the complexity and demands on time involved in implementing a new adapter type. However, it is necessary to understand this existing architecture in order to effectively use the provided building blocks. The following is intended as a guide to developing new adapters to extend the functionality of nb5.
```

Comment:
Possible rewording at point of: "quick a process".
Where 'nb' change to 'nb5'.

Reply:
Wording changed to "...makes the process of implementing new adapters as seamless and quick as possible". nb5 --> nb.